src/compas_rhino/scene/meshobject.py

```diff
@@ -149,16 +164,8 @@ def clear_facelabels(self):
     # draw
     # ==========================================================================
 
-    def draw(self, color=None, vertexcolors=None, facecolors=None, disjoint=False):
-        """Draw the mesh as a consolidated RhinoMesh.
-
-        Parameters
-        ----------
-        color : tuple[int, int, int], optional
-            The color of the mesh.
-            Default is the value of :attr:`MeshObject.default_color`.
-        disjoint : bool, optional
-            If True, draw the faces of the mesh with disjoint vertices.
+    def draw(self):
+        """Draw the mesh or its components in Rhino.
```

Comment:
removed here the optional inputs, the rational is that no one should call `obj.draw()` directly, one is supposed to always to `scene.draw()` instead. These options are either passed in at `scene.add()` or edited through the object.